ChangeLog.md

```diff
@@ -1,4 +1,10 @@
 # Branch Android SDK change log
+- v4.1.1
+  * _*Master Release*_ - October 7, 2019
+  * Support for Fire OS
+  * Do not collect advertising ID when limit ad tracking is enabled
```

Comment:
Can you update this to "Do not collect advertising ID when limit ad tracking is enabled after the app was initialized"? I want to be clear in the notes that it was set to not collect that before, but there was a bug with the feature caching that setting.